game/scripts/vscripts/items/item_greater_power_treads.lua

```diff
@@ -0,0 +1,299 @@
+				projSpeed = 950
+			end
+			
+			--
+			-- find all appropriate targets around the initial target
+			local units = FindUnitsInRadius(
+				parentTeam,
+				targetOrigin,
+				nil,
+				radius,
+				targetTeam,
+				targetType,
+				targetFlags,
+				FIND_ANY_ORDER,
+				false
+			)
+			
+			-- remove the initial target from the list
+			for k, unit in pairs( units ) do
+				if unit == target then
+					table.remove( units, k )
+					break
+				end
+			end
+			
+			-- fire a projectile at all targets
+			for k, unit in pairs( units ) do
+				ProjectileManager:CreateTrackingProjectile( {
+					Target = unit,
+					Source = target,
+					Ability = spell,
+					EffectName = projName,
+					iMoveSpeed = projSpeed,
+					iSourceAttachment = DOTA_PROJECTILE_ATTACHMENT_HITLOCATION,
+					bProvidesVision = false,
+					
+					ExtraData = {
+						damage = damage,
+					},
+				} )
+			end
+		end
+	end
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_power_treads:GetModifierMoveSpeedBonus_Special_Boots( event )
+	local spell = self:GetAbility()
+	
+	return spell:GetSpecialValueFor( "bonus_movement_speed" )
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_power_treads:GetModifierAttackSpeedBonus_Constant( event )
+	local spell = self:GetAbility()
+	
+	return spell:GetSpecialValueFor( "bonus_attack_speed" )
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_power_treads:GetModifierBonusStats_Strength( event )
+	local spell = self:GetAbility()
+	local attribute = self:GetStackCount() or DOTA_ATTRIBUTE_STRENGTH
+	
+	if attribute == DOTA_ATTRIBUTE_STRENGTH then
+		return spell:GetSpecialValueFor( "bonus_stat" )
+	end
+	
+	return 0
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_power_treads:GetModifierBonusStats_Agility( event )
+	local spell = self:GetAbility()
+	local attribute = self:GetStackCount() or DOTA_ATTRIBUTE_STRENGTH
+	
+	if attribute == DOTA_ATTRIBUTE_AGILITY then
+		return spell:GetSpecialValueFor( "bonus_stat" )
+	end
+	
+	return 0
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_power_treads:GetModifierBonusStats_Intellect( event )
+	local spell = self:GetAbility()
+	local attribute = self:GetStackCount() or DOTA_ATTRIBUTE_STRENGTH
+	
+	if attribute == DOTA_ATTRIBUTE_INTELLECT then
+		return spell:GetSpecialValueFor( "bonus_stat" )
+	end
+	
+	return 0
+end
+
+--------------------------------------------------------------------------------
+
+item_greater_power_treads_2 = item_greater_power_treads
+item_greater_power_treads_3 = item_greater_power_treads
+item_greater_power_treads_4 = item_greater_power_treads
+item_greater_power_treads_5 = item_greater_power_treads
```

Comment:
you should use an editor that supports editorconfig so that it automatically avoids formatting errors like this